Sourcery/Templates/Diffable.stencil

```diff
@@ -2,12 +2,12 @@
 extension {{ type.name }}{% if not type.supertype.implements.AutoDiffable %}: Diffable{% endif %} {
     {% if type.based.Type %}override {% endif %}func diffAgainst(_ object: Any?) -> DiffableResult {
         let results = DiffableResult()
-        {% if type.storedVariables.count %}guard let rhs = object as? {{ type.name }} else {
+        {% for variable in type.storedVariables %}{% if not variable.annotations.skipEquality %}{% if forloop.first %}guard let rhs = object as? {{ type.name }} else {
```

Comment:
Then it will produce a warning for unused property, that's why I changed that in the first place.